src/transformers/configuration_utils.py

```diff
@@ -467,7 +467,12 @@ def save_pretrained(self, save_directory: Union[str, os.PathLike], push_to_hub:
             )
 
     @classmethod
-    def from_pretrained(cls, pretrained_model_name_or_path: Union[str, os.PathLike], **kwargs) -> "PretrainedConfig":
+    def from_pretrained(
+        cls,
+        pretrained_model_name_or_path: Union[str, os.PathLike],
+        token: Optional[Union[str, bool]] = None,
```

Comment:
For `PretrainedConfig`, I guess I have  to deal with all arguments that we want to be explicit **in this PR**.